RNA-seq/00a-reproducibility_cmdline.md

```diff
@@ -68,26 +68,58 @@ do
   fastqc ${filename} -o <QC_DIRECTORY>/${name}
 done
 ```
-where `QC_DIRECTORY is the directory to hold the QC reports and FASTQ_DIRECTORY is the directory that contains the `fastq` files.
+
+where `QC_DIRECTORY` is the directory to hold the QC reports and FASTQ_DIRECTORY is the directory that contains the `fastq` files.
 The `<` and `>` indicate that this is to be replaced by you, the user, rather than run as is.
```

Comment:
I end up calling these <FILL_IN_THE_BLANKS> all the time.

Reply:
This is designed to be reference material. So I think the way this is currently written, to be more explicit, is appropriate.